endPoint/h2HttpHostEndPoint.js

```diff
@@ -123,7 +133,14 @@ export default class H2HttpHostEndPoint extends SecureHttpHostEndPoint {
               headers[":method"]
             );
             headerList[":status"] = code;
-            stream.respond(headerList);
+            stream.respond({
+              ...headerList,
+              "Strict-Transport-Security":
+                "max-age=15552000; includeSubDomains; preload",
+              "X-Content-Type-Options": "nosniff",
+              "X-Frame-Options": "DENY",
+              "X-XSS-Protection": "1; mode=block",
+            });
```

Comment:
Adding this header in code is not good idea!
I think this headers must set/not-set in each request process business separately...